### PR TITLE
Adds Zika references and Zibra primers

### PR DESCRIPTION
**Pull request by aavilaherrera**

### CODE REVIEW

File: mappgene/data/extra_files/ivar/Primers for amplicon-based sequencing - ZIKV - Asia_America - 400bp.tsv
Comment (khyox) [inline mappgene/data/extra_files/ivar/Primers for amplicon-based sequencing - ZIKV - Asia_America - 400bp.tsv:1]: I am not against spaces in filenames (or in names, surnames, you know!), on the contrary, but this one with 9 spaces is a bit scary! ;D

File: README.md
Comment (khyox) [inline README.md:1]: Great documentation! 🥇 Very useful the "Notes" column in the table for quickly understanding the source of the reference (e.g., "Zika virus strain H/PF/2013, complete genome." corresponding to French polynesia outbreak in 2013)! Thanks!

**Review by khyox — APPROVED** [review]
Great additions to the mappgene with this comprehensive expansion to Zika! Many thanks!!

**Comment by aavilaherrera** [comment]
Agreed! Thankfully, it is not used except for as a reference data source. The primer names with backslash+star are also scary. Just in time for Halloween! 🎃

From: Jose Manuel Martí ***@***.***>
Reply-To: LLNL/mappgene ***@***.***>
Date: Monday, October 16, 2023 at 11:21 PM
To: LLNL/mappgene ***@***.***>
Cc: "Avila-Herrera, Aram" ***@***.***>, Author ***@***.***>
Subject: Re: [LLNL/mappgene] Adds Zika references and Zibra primers (PR #16)


@khyox commented on this pull request.

________________________________

On mappgene/data/extra_files/ivar/Primers for amplicon-based sequencing - ZIKV - Asia_America - 400bp.tsv<https://urldefense.us/v3/__https:/github.com/LLNL/mappgene/pull/16*discussion_r1361560400__;Iw!!G2kpM7uM-TzIFchu!zjfrpBfKVSCtiaYFW0wqcBB1-SGUglEHqu7UdZC7C9MUy1xIXJIobNPVeeofyKZcOJ1udNQfnT_6nsp8qfobz3zkzpiC0g$>:

I am not against spaces in filenames (or in names, surnames, you know!), on the contrary, but this one with 9 spaces is a bit scary! ;D

**Comment by khyox** [comment]
Yeah, the primer names are worse than any Halloween costume, hahaha! 🎃🎃🎃

**Pull request merged**